src/main/navigation/navigation_private.h

```diff
@@ -176,41 +176,82 @@ typedef enum {
     NAV_FSM_EVENT_COUNT,
 } navigationFSMEvent_t;
 
+// This enum is used to keep values in blackbox logs stable, so we can
+// freely change navigationFSMState_t.
```

Comment:
I don't like the name `PUBLIC_STATE`, same naming as in box modes seems more reasonable - `PERSISTENT_ID`

Reply:
Roger that